cmd/loki/loki-docker-config.yaml

```diff
@@ -17,7 +17,7 @@ common:
 schema_config:
   configs:
     - from: 2020-10-24
-      store: boltdb-shipper
+      store: tsdb
       object_store: filesystem
       schema: v11
```

Comment:
Does the date and schema need to be updated too? 
I'm not sure about `from` but i know tsdb is using `v12`